Lib/base64.py

```diff
@@ -576,6 +576,27 @@ def decodebytes(s):
     return binascii.a2b_base64(s)
 
 
+# Use accelerated implementations of originally pure-Python parts if possible.
+try:
+    from _base64 import (_a85encode, _a85decode, _b85encode,
+                         _b85decode, _z85encode, _z85decode)
+    from functools import update_wrapper
```

Comment:
Functools is an expensive import, I would copy the relative parts of `update_wrapper()` locally.